src/test/resources/profiles/mappingProfile.json

```diff
@@ -11,7 +11,7 @@
       "fieldId": "electronicAccess.linkText",
       "path": "Json path for electronicAccess[*].linkText",
```

Comment:
Is there a reason for using the text in path field? I remember it was not supported in sample file because of raml issues. But we should probably be using proper format in tests

Reply:
Updated